test/source/specs/base/concurrency/task.js

```diff
@@ -574,5 +575,14 @@ describe('Data.Task', () => {
       const task = convertedFn('test');
       await task.run().promise().catch(e => $ASSERT(e === error));
     });
+
+    it('Should allow rejecting more than once', async () => {
```

Comment:
Following [corresponding `future` test case](https://github.com/origamitower/folktale/blob/master/test/source/specs/base/concurrency/future.js#L72) and my [own review](https://github.com/origamitower/folktale/pull/216#discussion_r247230854).

```js
it('shouldnt allow resolving more than once', ...)
```